Update PSDesiredStateConfiguration.md to improve clarity (#314)
MicrosoftDocs/PowerShell-Docs-DSC: dsc/dsc-3.0/PSDesiredStateConfiguration/PSDesiredStateConfiguration.md

# PSDesiredStateConfiguration Module

## Description

The PSDesiredStateConfiguration module is a module for PowerShell Desired State Configuration
(PSDSC). While this module still exists for invoking both script-based and

> class-based DSC resources, it is only used in Microsoft Desired State Configuration (DSC) to execute

Reviewer: ```suggestion
class-based DSC resources, it's only used in Microsoft Desired State Configuration (DSC) to execute
```

> The PSDesiredStateConfiguration module is a module for PowerShell Desired State Configuration
> (PSDSC). While this module still exists for invoking both script-based and
> class-based DSC resources, it is only used in Microsoft Desired State Configuration (DSC) to execute
> script-based DSC resources through the Microsoft.Windows/WindowsPowerShell adapter.
> 
> > [!NOTE]
> > Only PSDesiredStateConfiguration v1.1 of the module is used in the
> > Microsoft.Windows/WindowsPowerShell adapter through Microsoft DSC.

Reviewer: ```suggestion
(PSDSC). Each major version of this module supports different use cases:

- v1.1 is only supported for Windows PowerShell. You can use it to invoke PSDSC resources and to
  author and apply PSDSC configurations.
- v2 is only supported for PowerShell. You can use it to invoke PSDSC resources and to compile
  PSDSC configurations. It's intended use is to support Azure Machine Configuration.
- v3 (preview) is only supported for PowerShell. You can use it to invoke PSDSC resources. It's
  intended use is for invoking PSDSC resources on Linux machines with Azure Machine Configuration.

Microsoft Desired State Configuration (DSC) enables you to manage PSDSC resources through two
adapters:

- The `Microsoft.Windows/WindowsPowerShell` adapter enables you to invoke any PSDSC resource usable
  with **PSDesiredStateConfiguration** version 1.1 in Windows PowerShell.
- The `Microsoft.DSC/PowerShell` adapter enables you to invoke any class-based PSDSC resource in
  PowerShell. It doesn't rely on the **PSDesiredStateConfiguration** module.
```
